internal/controllers/utils.go

```diff
@@ -58,6 +74,7 @@ func (r *ipPolicyResolver) resolveIPPolicyNamesorIds(ctx context.Context, namesp
 		if err := r.client.Get(ctx, types.NamespacedName{Name: nameOrId, Namespace: namespace}, policy); err != nil {
 			if client.IgnoreNotFound(err) == nil {
 				m[nameOrId] = true // assume it's an ID
+				continue
```

Comment:
this actually allow us to use `ipp_`, e.g. preexisting policies not managed in k8s.

Reply:
nice find!